Lib/asyncio/coroutines.py

```diff
@@ -9,16 +9,14 @@
 import traceback
 import types
 
+from collections.abc import Awaitable, Coroutine
+
 from . import constants
 from . import events
 from . import base_futures
 from .log import logger
 
 
-# Opcode of "yield from" instruction
-_YIELD_FROM = opcode.opmap['YIELD_FROM']
```

Comment:
Is opcode still used?

Reply:
No. It was used for making a workaround for bug already fixed in Python 3.4